virt/virtual_machines/virtual_disks/virt-creating-and-using-boot-sources.adoc

```diff
@@ -26,8 +26,9 @@ include::modules/virt-creating-a-custom-image-boot-source-web.adoc[leveloffset=+
 [role="_additional-resources"]
 == Additional resources
 * xref:../../../virt/vm_templates/virt-creating-vm-template.adoc#virt-create-vms[Creating virtual machine templates]
-* link:https://www.openshift.com/blog/creating-windows-virtual-machines-from-existing-images-with-openshift-virtualization[Creating a Microsoft Windows boot source from a cloud image]
-* link:https://www.openshift.com/blog/customizing-virtual-machine-templates-in-openshift-1[Customizing existing Microsoft Windows boot sources in {product-title}]
-* link:https://www.openshift.com/blog/creating-default-os-images-to-auto-clone-in-openshift-virtualization[Setting a PVC as a boot source for a Microsoft Windows template using the CLI]
+* xref:../../../virt/virtual_machines/advanced_vm_management/virt-automatic-bootsource-updates.adoc#virt-automatic-bootsource-updates[Automatic importing and updating of pre-defined boot sources]
+* link:https://www.openshift.com/blog/creating-windows-virtual-machines-from-existing-images-with-openshift-virtualization[Creating a Windows boot source from a cloud image]
```

Comment:
I'm not sure we should keep linking to these old blogs. They have to be reviewed at some point, probably by Chandler.

Reply:
I had a couple of conversations with Chandler when we were refining the Getting Started page content, but I can have him re-review them.

Reply:
I also chatted with him about the blogs. I think that's why, in the end, we did not include the links.